File: Kotlin/app/src/main/java/ch/liip/appbuilders2019/api/Todo.kt
package ch.liip.appbuilders2019.api

import kotlinx.serialization.Serializable

@Serializable
data class Todo(
    var id: Int,
    var title: String,
    var completed: Boolean
)

fun Todo.summary(): String {
    val doneValue = if (completed) "✅" else "❌"
    return "$doneValue $title"
}

File: Swift/AppBuilders2019/Todo.swift
import Foundation

struct Todo: Codable {
    var id: Int
    var title: String
    var completed: Bool
}

extension Todo {
    func summary() -> String {
        let doneValue = completed ? "✅" : "❌"
        return "\(doneValue) \(title)"
    }
}
